packages/nimbus/src/components/icon/icon.mdx

```diff
@@ -14,73 +14,192 @@ tags:
+    <path d="M16.8127 35.0107V20.3334C16.8127 19.8948 17.2866 19.622 17.6664 19.84L30.1638 27.0559C30.6377 27.3301 30.6377 28.0141 30.1638 28.2882L17.6664 35.5041C17.2866 35.7234 16.8127 35.4493 16.8127 35.0107Z" fill="#0BBFBF"/>
+  </svg>`;
+
+  return (
+    <Stack direction="row" gap="400" alignItems="center">
+      <Icon size="xs"><InlineSvg data={blackLogo} /></Icon>
+      <Icon size="xs"><InlineSvg data={colorLogo} /></Icon>
+    </Stack>
+  );
+};
 ```
 
-### Predefined sizes
+#### Predefined sizes
 
-The `Icon` component comes with a set of predefined sizes that can be applied using the `size` prop.
+The `Icon` component comes with a set of predefined sizes that can be applied
+using the `size` prop.
 
 ```jsx-live
 const App = () => (
   <Stack direction="row" gap="400" alignItems="center">
-    <Icon as={Icons.Bathtub} size="2xs" />
-    <Icon as={Icons.Bathtub} size="xs" />
-    <Icon as={Icons.Bathtub} size="sm" />
-    <Icon as={Icons.Bathtub} size="md" />
-    <Icon as={Icons.Bathtub} size="lg" />
-    <Icon as={Icons.Bathtub} size="xl" />
+    <Icon size="2xs"><Icons.SentimentSatisfied /></Icon>
+    <Icon size="xs"><Icons.SentimentSatisfied /></Icon>
+    <Icon size="sm"><Icons.SentimentSatisfied /></Icon>
+    <Icon size="md"><Icons.SentimentSatisfied /></Icon>
+    <Icon size="lg"><Icons.SentimentSatisfied /></Icon>
+    <Icon size="xl"><Icons.SentimentSatisfied /></Icon>
   </Stack>
 );
 ```
 
-### Custom size
+#### Custom size
 
 You can apply a custom size to the `Icon` using the `boxSize` style prop.
 
 ```jsx-live
-const App = () => <Icon as={Icons.Bathtub} boxSize="3200" />;
+const App = () => <Icon boxSize="3200"><Icons.SentimentSatisfied /></Icon>;
 ```
 
-### Custom color
+## Guidelines
+
+Icon guidelines establish principles for designing and using consistent,
+recognizable, and accessible graphical symbols. They ensure icons clearly convey
+meaning, maintain visual harmony, and function effectively across various
+contexts and screen sizes while adhering to accessibility standards.
+
+### Best practices
+
+- **Clarity & Recognition:**
+  - **Be intuitive:** Design icons that are instantly recognizable and clearly
+    convey their meaning without needing extensive explanation.
+  - **Supplement with text:** If an icon's meaning might be ambiguous, pair it
+    with a text label, especially for critical actions.
+- **Consistency & Style:**
+  - **Maintain visual harmony:** Ensure all icons adhere to a consistent visual
+    style (e.g., line weight, fill vs. outline, corner radius, perspective).
+  - **Use consistently:** Always use the same icon to represent the same action
+    or concept throughout the entire product.
+  - **Ensure proper alignment:** Pay attention to pixel grid alignment for
+    crispness, especially at smaller sizes.
+- **Usability & Interaction:**
+  - **Provide clear states:** Design distinct visual states for interactive
+    icons (e.g., hover, focus, active, disabled).
```

Comment:
Should we add that in the vast majority of cases you should use the `IconButton` for interactive icons?